config/externalname.go

```diff
@@ -2233,10 +2233,10 @@ var ExternalNameConfigs = map[string]config.ExternalName{
 	//
 	// Network Firewall Policies can be imported using their ARN
 	// Example: arn:aws:network-firewall:us-west-1:123456789012:firewall-policy/example
-	"aws_networkfirewall_firewall_policy": config.TemplatedStringAsIdentifier("name", "arn:aws:network-firewall:{{ .parameters.region }}:{{ .setup.client_metadata.account_id }}:firewall-policy/{{ .external_name }}"),
+	"aws_networkfirewall_firewall_policy": config.IdentifierFromProvider,
```

Comment:
I've successfully verified the following with the import scenario:
```suggestion
	"aws_networkfirewall_firewall_policy": config.TemplatedStringAsIdentifier("name", "arn:aws:network-firewall:{{ .setup.configuration.region }}:{{ .setup.client_metadata.account_id }}:firewall-policy/{{ .external_name }}"),
```

Why I'm suggesting this is because we keep the API normalized as discussed in some depth [here](https://github.com/upbound/provider-aws/pull/661#discussion_r1165788746).